mods/blocks/plank.lua

```diff
@@ -123,6 +191,44 @@ minetest.register_craft({
 	}
 })
 
+minetest.register_craft({
+	output = "blocks:bonewood",
+	recipe = {
+		{"flora:bone_bush_stem"},
+	}
+})
+
+minetest.register_craft({
+	output = "blocks:firewood",
+	recipe = {
+		{"blocks:firebranches", "blocks:firebranches"},
+		{"blocks:firebranches", "blocks:firebranches"},
+	}
+})
+
+minetest.register_craft({
+	output = "blocks:mushroom_wood",
```

Comment:
sure, but it will make things even more confusing

Reply:
right, but i think currently the fact that it is converted from/to two different type of sticks is even more confusing

Reply:
Actually, the wood isn't being crafted from the sticks so this works out as is

Reply:
Ah right, but this still presents another question as to the usage of mushroom sticks then.
Considering mushroom sticks have the same exact function as normal sticks; they are part of redundancy now?
Since is already possible to get sticks from converting mycelia into wood and then into normal sticks.

Reply:
Not sure what to do about this now, I completely forgot that sticks are crafted from group:wood (and I have to have mushroom wood be a group:wood). Maybe remove mushroom sticks completely?

Reply:
I think for now yeah, remove them.
Leave the texture though, its an interesting one, maybe it will find some use along with the farming changes.